<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDialog</class>
 <widget class="QDialog" name="ItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item</string>
  </property>
  <widget class="QLabel" name="lblPrice">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLoginID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item Code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDescription">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtItemCode">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrice">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtDescription</tabstop>
  <tabstop>txtItemCode</tabstop>
  <tabstop>txtPrice</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
